### PR TITLE
Random (IDE lints) fixes

### PR DESCRIPTION
**Pull request by stanislav-tkach**
After seeing the https://github.com/mintlayer/mintlayer-core/pull/768 pull request, I have decided to try the analysis embedded in my IDE (CLion with intellij-rust plugin). Surprisingly it hasn't found many typos or grammar errors, but it gives some helpful lints instead. For example "using assert_eq/assert_ne" , "redundant semicolon" or "unnecessary qualified path". The most important thing that it found was the unused [history_iteration.rs](https://github.com/mintlayer/mintlayer-core/blob/master/chainstate/test-suite/src/tests/history_iteration.rs) file.

### CODE REVIEW

File: utils/src/blockuntilzero.rs [hunk 1]
```diff
@@ -56,7 +56,6 @@ where
         self.value.load(Ordering::Acquire)
     }
 
-    #[must_use = "CountTracker is useless without holding its object"]
```
Comment (stanislav-tkach) [inline utils/src/blockuntilzero.rs:59]: This isn't an error: the `CountTracker` type already has the `must_use` attribute, so this is redundant.

File: utils/src/counttracker.rs [hunk 2]
```diff
@@ -33,7 +33,6 @@ where
     <T as Atomic>::Type: One,
     <T as Atomic>::Type: Zero,
 {
-    #[must_use = "CountTracker is useless without holding its object"]
```
Comment (stanislav-tkach) [inline utils/src/counttracker.rs:36]: Same here.

File: common/src/chain/config/builder.rs [hunk 3]
```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use super::emission_schedule::{self, *};
+use super::emission_schedule::*;
```
Comment (TheQuantumPhysicist) [inline common/src/chain/config/builder.rs:16]: I'm a big opponent of this kind of imports... very confusing and makes it hard to read files.

**Review by TheQuantumPhysicist — APPROVED** [review]

File: common/src/chain/config/builder.rs (hunk 3 above)
Comment (stanislav-tkach) [reply common/src/chain/config/builder.rs]: I agree, fixed.

**Review by pavel-kokolemin — APPROVED** [review]

**Review by muursh — APPROVED** [review]

**Pull request merged**